src/components/20-molecules/file-upload/index.js

```diff
@@ -206,26 +240,37 @@ class AXAFileUpload extends LitElement {
   }
 
   async addFiles(droppedFiles, removeGlobalMessage) {
-    this.showAddMoreInputFile = true;
+    // generate id to match orignal files with compressed one if this.accessOriginalFiles is set
+    const droppedFilesWithID = droppedFiles.map(file => {
+      file.id = Math.random()
+        .toString(36)
+        .substr(2, 9);
```

Comment:
Dont we have a helper function to create a unique id? I thought we need that at another component too.

Reply:
nice finding! yes we have one. it's called create refid an look pretty similar.
```
export default (prefix = '') =>
  `${prefix}${Math.random()
    .toString(36)
    .slice(2)}`;

```

Reply:
Going to use that.